data/application.go

```diff
@@ -45,5 +43,15 @@ var (
 			Path: "example-1",
 		},
 	}
-	ApplicationSamples = []api.Application{Minimal, PathfinderGit, BookServer, TackleTestappPublic, UploadBinary, CustomerTomcatLegacy}
+
+	// Binary applications
+	TackleTestappPublicBinary = api.Application{
+		Name: "Tackle Testapp public binary",
+		Binary: "mvn://io.konveyor.demo:customers-tomcat:0.0.1-SNAPSHOT:war",	// To be downloaded from maven repository
```

Comment:
The `mvn://` is needed since https://github.com/konveyor/tackle2-hub/pull/668 (having a generic addon and java logic is just within java-external-provider in analyzer-lsp repo).

With missing `mvn://` prefix, analyzer doesn't download (also not failing since it cares only about binaries string with the mvn prefix https://github.com/konveyor/analyzer-lsp/blob/main/external-providers/java-external-provider/pkg/java_external_provider/provider.go#L246).

I'm not sure if this was documented somewhere or the java provider should assume mvn prefix (likely not).